frontend/stores/organization.ts

```diff
@@ -188,34 +188,38 @@ export const useOrganizationStore = defineStore("organization", {
           (text) => text.data._value.results
         )[0] as unknown as OrganizationText[];
 
+        console.log(`Hereee ${JSON.stringify(orgTextsData)}`);
+
         // Order texts based on the returned organizations.
-        const orgIDs = orgs._value.map((o) => o.id);
+        const orgIds = orgs._value.map((o) => o.id);
         const sortedOrgTextsData: OrganizationText[] = [];
-        for (const id of orgIDs) {
+        for (const id of orgIds) {
           for (const text of orgTextsData) {
-            if (text.orgID === id) {
+            if (text.orgId === id) {
               sortedOrgTextsData.push(text);
             }
           }
         }
 
+        console.log(`Hereee ${JSON.stringify(sortedOrgTextsData)}`);
```

Comment:
Is this for manual debugging?  Does it need to stay in the PR?

Reply:
Ya that's not intentional. Will be removed. I'm debugging to try to return organizations on the `/organizations` route, and this the check here is how I'm finding that we're still getting back `snake_case`.